numbers-csharp/ConvertString2Int/ConvertString2Int/CustomConvert.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConvertString2Int
+{
+    public static class CustomConvert
+    {
+        public static int Parse(string strVal)
+        {
+            int num = 0;
+            for (int i = 0; i < strVal.Length; i++)
+            {
+                num = num * 10 + (strVal[i] - '0');
+            }
+            return num;
```

Comment:
Newlines after the variable definition and before the return statement. use var instead of int for variable definition.